src/modules/flow/si114x/si114x.json

```diff
@@ -0,0 +1,65 @@
+{
+  "$schema": "http://solettaproject.github.io/soletta/schemas/node-type-genspec.schema",
+  "name": "si114x",
+  "meta": {
+    "author": "Intel Corporation",
+    "license": "BSD 3-Clause",
+    "version": "1"
+  },
+  "types": [
+    {
+      "category": "output/sw",
+      "description": "Provides ambient uv light reading, datasheet at: https://www.silabs.com/Support%20Documents/TechnicalDocs/Si1145-46-47.pdf",
+      "methods": {
+        "close": "si114x_close",
+        "open": "si114x_open",
+        "process": "si114x_process"
+      },
+      "name": "light-sensor/si114x",
+      "options": {
+        "members": [
+          {
+            "data_type": "int",
+            "description": "Sensor BUS",
+            "name": "bus"
+          },
+          {
+            "data_type": "int",
+            "description": "Sensor Address",
+            "name": "address"
+          },
+          {
+            "data_type": "int",
+            "description": "Sensor reading speed",
+            "name": "speed"
+          },
+          {
+            "data_type": "int",
+            "description": "uv-calibration",
```

Comment:
You could set the default one here, don't you think? But please, note my first comment about this.